x-pack/libbeat/management/config.go

```diff
@@ -98,7 +98,8 @@ func defaultConfig() *Config {
 		},
 		Blacklist: ConfigBlacklistSettings{
 			Patterns: map[string]string{
-				"output": "console|file",
+				"output":      "console|file",
+				".processors": "script",
```

Comment:
@michalpristas Do you really need to have a dot prefix? I though only definining the following ".processors": "script" would be catch?

Reply:
it will not catch it as blacklist is looking for a prefix. so if processor is defined within filebeat.input it  requires filebeat.input.processors

other option would be to get rid of dot prefix and scan only for index, index("filebeat.input.processors", "processors")
this will yield true but i think it is not so deterministic as saying `.processors` with a conscious choice of having it wherever not just on top of the block

Reply:
strategy LGTM.